docs/src/contributing_federation_development.md

```diff
@@ -5,14 +5,12 @@
 If you don't have a local clone of the Lemmy repo yet, just run the following command:
 
 ```bash
-git clone https://github.com/LemmyNet/lemmy -b federation
+git clone https://github.com/LemmyNet/lemmy
 ```
 
 If you already have the Lemmy repo cloned, you need to add a new remote:
```

Comment:
This part isnt needed anymore, as its all in the normal master branch.

Reply:
The plus has it removed :)

Reply:
Not sure what you mean, you are still adding a new remote which isnt needed.

Reply:
Ohh okay I see what you mean now, yes I can remove this section.